src/pages/Filing/UpdateFinancialProfile/AdditionalDetails.tsx

```diff
@@ -0,0 +1,35 @@
+// TODO: vv Revisit these exceptions vv
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import LabelOptional from 'components/LabelOptional';
+import SectionIntro from 'components/SectionIntro';
+import { Label, TextArea, WellContainer } from 'design-system-react';
+import type { JSXElement } from 'design-system-react/dist/types/jsxElement';
+import { SectionWrapper } from './UpdateIdentifyingInformation';
+
+function AdditionalDetails({ register }: { register: any }): JSXElement {
+  return (
+    <SectionWrapper>
+      <SectionIntro heading='Provide any additional details'>
+        Do not include personal identifiable information (PII) or other
+        sensitive information in the field below, such as your personal address,
+        Social Security number, or passwords.
+      </SectionIntro>
+      <WellContainer className='u-mt30'>
+        <Label htmlFor='stuff'>
+          Additional details
+          <LabelOptional />
+        </Label>
+        <TextArea
```

Comment:
>  Bug?: User can resize the Additional details TextArea outside of grey bounding box. Do we need to restrict this?

For future work: maybe we can slap in a css rule here `resize: vertical` to prevent this?